README.md

```
@@ -1,33 +1,34 @@
-## Peer Dependency
+### Peer Dependency
 This package relies on @nicolawealth/ioc as a peer dependency. It will not be bundled with rate_limit, so you need to install it in your project:
 `npm install @nicolawealth/ioc`
 
-# Compatibility Notes
+## Compatibility Notes
 The rate_limit package ships with two build formats:
 
-### ESM (Modern)
-#### (dist/index.modern.js)
+#### ESM (Modern)
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)